src/Sentry/Internal/DsnLocator.cs

```diff
@@ -19,5 +19,8 @@ internal static string FindDsnStringOrDisable(Assembly? asm = null)
         /// <returns>DSN string or null if none found.</returns>
         internal static string? FindDsn(Assembly? asm = null)
             => (asm ?? Assembly.GetEntryAssembly())?.GetCustomAttribute<DsnAttribute>()?.Dsn;
+
+        internal static string? Resolve() =>
+            Environment.GetEnvironmentVariable(Constants.DsnEnvironmentVariable) ?? FindDsn();
```

Comment:
Note: `FindDsnStringOrDisable` is now completely unused. I'm not sure what its purpose is at this point (or was, for that matter, if the current behavior was expected before).